src/server/execute_process_queue.ts

```diff
@@ -0,0 +1,29 @@
+import * as spawn from "child_process"
+import * as winston from "winston"
+import {ProcessQueue} from "./process_queue"
+
+export class ExecuteProcessQueue extends ProcessQueue {
+
+    async run(data: string) {
+        return this.queue.add(async () => {
+            return new Promise<string>((resolve, reject) => {
+                const child = spawn.fork(`./src/actions/actions_process.ts`)
```

Comment:
I like the rename of this file but I meant renaming `src/actions/actions_process.ts` – perhaps to `src/execute_worker.ts`

Reply:
My other comment got outdated by your renaming but before the rename I proposed this https://github.com/looker/actions/pull/192#discussion_r215060258